src/helpers/helpers.dom.js

```diff
@@ -56,7 +57,7 @@ function getConstraintDimension(domNode, maxStyle, percentageProperty) {
 			hasCContainer ? parseMaxStyle(constrainedContainer, parentNode, percentageProperty) : infinity);
 	}
 
-	return 'none';
+	return undefined;
```

Comment:
I think this like can be removed. Babel might be smart enough to remove it too though.

Reply:
done